products/revenue_analytics/frontend/settings/EventConfiguration.tsx

```diff
@@ -9,7 +9,7 @@ import { Tooltip } from 'lib/lemon-ui/Tooltip'
 
 import { RevenueAnalyticsEventItem } from '~/queries/schema/schema-general'
 
-import { CurrencyDropdown } from './CurrencyDropdown'
+import { CurrencyDropdown } from '../../../../frontend/src/lib/components/BaseCurrency/CurrencyDropdown'
```

Comment:
You can do this instead

```suggestion
import { CurrencyDropdown } from 'lib/components/BaseCurrency/CurrencyDropdown'
```